DNN-OpenCV-Classification-with-Java/README.md

```diff
@@ -0,0 +1,190 @@
+```
+
+5. Compile OpenCV. Create and enter a build directory:
+
+```bash
+cd ~/opencv
+mkdir build
+```
+
+6. Run CMake to configure the OpenCV build. Don't forget to set the right pass to the ``PYTHON_EXECUTABLE``:
+
+```bash
+cmake -D CMAKE_BUILD_TYPE=RELEASE \
+ -D CMAKE_INSTALL_PREFIX=/usr/local \
+ -D INSTALL_PYTHON_EXAMPLES=OFF \
+ -D INSTALL_C_EXAMPLES=OFF \
+ -D OPENCV_ENABLE_NONFREE=ON \
+ -D OPENCV_EXTRA_MODULES_PATH=~/opencv_contrib/modules \
+ -D PYTHON_EXECUTABLE=~/env/bin/python3 \
+ -D BUILD_SHARED_LIBRARY=OFF \
+ -D BUILD_TESTS=OFF \
+ -D BUILD_PERF_TESTS=OFF \
+ -D BUILD_EXAMPLES=ON ..
+```
+
+If you want to configure the build with some specific Java version, please, add the following fields, verifying the paths:
+
+```bash
+ -D JAVA_AWT_INCLUDE_PATH=/usr/lib/jvm/java-1.x.x-openjdk-amd64/include \
+ -D JAVA_AWT_LIBRARY=/usr/lib/jvm/java-1.x.x-openjdk-amd64/lib/libawt.so \
+ -D JAVA_INCLUDE_PATH=/usr/lib/jvm/java-1.x.x-openjdk-amd64/include \
+ -D JAVA_INCLUDE_PATH2=/usr/lib/jvm/java-1.x.x-openjdk-amd64/include/linux \
+ -D JAVA_JVM_LIBRARY=/usr/lib/jvm/java-1.x.x-openjdk-amd64/include/jni.h \
+```
+
+7. Check the output and make sure that everything is set correctly. After that we're ready to build it with:
+
+```bash
+make -j8
+```
+
+Make sure, you didn't get any errors. In case of successful completion you will find the following files in the ``build`` directory:
+* ``bin/opencv-430.jar``
+* ``lib/libopencv_java430.so``
+* ``lib/python3/cv2.cpython-37m-x86_64-linux-gnu.so``
+
+Then run the following command:
+
+```bash
+sudo ldconfig
+```
+
+which creates the necessary links and cache to our freshly built shared library.
+
+Put ``lib/python3/cv2.cpython-37m-x86_64-linux-gnu.so`` into the virtual environment installed packages:
+
+```bash
+cp lib/python3/cv2.cpython-36m-x86_64-linux-gnu.so ~/env/lib/python3.7/site-packages/cv2.so
+```
+
+The last step is to put ``~/opencv/build/lib/libopencv_java430.so`` into the ``/usr/lib`` directory:
+
+```bash
+sudo cp lib/libopencv_java430.so /usr/lib
+```
+
+For Windows and macOS OpenCV Java build, please, follow the steps described in [Introduction to Java Development](https://docs.opencv.org/master/d9/d52/tutorial_java_dev_intro.html) or [Installing OpenCV for Java](https://opencv-java-tutorials.readthedocs.io/en/latest/01-installing-opencv-for-java.html).
+
+
+## Executing Model Conversion and Test Script
+The proposed for the experiments ``MobileNetV2ToOnnx.py`` script supports the ``--input_image`` key to customize the model conversion pipeline. It defines the full input image path, including its name - ``"coffee.jpg"`` by default.
+
+To run MobileNetV2 conversion case, please, choose one of the described below scenarios:
+
+* for the custom input image and running evaluation of the converted model:
+
+```bash
+python3 MobileNetV2Conversion.py --input_image "images/red-ceramic-mug.jpg"
+```
+
+* for the default input image:
+
+```bash
+python3 MobileNetV2Conversion.py
+```
```

Comment:
`MobileNetV2Conversion.py` does not exist. Please change it to `Mobilenetv2ToOnnx.py`